packages/rolldown/tests/validate-option.test.ts

```diff
@@ -51,3 +52,22 @@ test('validate output option', async () => {
       `)
   }
 })
+
+test('give a error for hoistTransitiveImports: true', async () => {
+  try {
+    const bundle = await rolldown({
+      input: './build-api/main.js',
+      cwd: import.meta.dirname,
+    })
+    await bundle.write({
+      // @ts-ignore  invalid value
+      hoistTransitiveImports: true
+    })
+    expect.unreachable()
+  } catch (error: any) {
+    expect(error.message).toMatchInlineSnapshot(`
+        "Failed validate output options.
+        - For the "hoistTransitiveImports". The 'true' value is not supported. "
+      `)
```

Comment:
@sapphi-red Give a error could be acceptable?

Reply:
Yes, it's ok.